src/client/components/MainFooter/MainFooter.tsx

```diff
@@ -24,71 +24,72 @@ interface FooterBlockProps {
 }
 
 function FooterBlock({ id, title, links, description }: FooterBlockProps) {
-  const titleRole = useDesktopScreen() ? 'columnheader' : 'button';
   return (
-    <Grid.Cell span={4} className={'page-footer-space-y '}>
-      <Heading
-        inverseColor={true}
-        level={4}
-        role={titleRole}
-        className="ams-mb--xs"
-      >
+    <Grid.Cell key={title} span={4} className="page-footer-space-y">
+      <Heading inverseColor level={4} className="ams-mb--xs">
         {title}
       </Heading>
       {!!description && getEl(description as AstNode)}
       {!!links.length && (
-        <UnorderedList markers={false}>
+        <LinkList>
           {links.map((link) => (
-            <LinkList key={link.title}>
-              <LinkList.Link
-                variant="inList"
-                onBackground={'dark'}
-                href={link.to}
-              >
-                {link.title}
-              </LinkList.Link>
-            </LinkList>
+            <LinkList.Link
+              key={link.to}
+              variant="inList"
+              onBackground="dark"
+              href={link.to}
+            >
+              {link.title}
+            </LinkList.Link>
           ))}
-        </UnorderedList>
+        </LinkList>
       )}
     </Grid.Cell>
   );
 }
-function getEl(astElement: AstNode | AstNode[], index: number = 0) {
-  if (Array.isArray(astElement) && astElement.length > 0) {
+
+function getEl(astElement: AstNode | AstNode[], i: number = 0) {
+  let index = i;
```

Comment:
Je re-assigned deze index niet toch?

Reply:
ja doe ik bewust...reference issues. Maar index is sowieso niet meer nodig

Reply:
ik heb nu useId gebruikt. Dat werkt goed. Het zou alleen een issues worden wanneer er elementen bij komen oid of de lijst verandert. Maar dat is niet het geval(dan zou je ze aan de AstNode van tevoren moeten toevoegen